apps/src/lib/ui/PopUpMenu.story.jsx

```diff
@@ -1,56 +1,62 @@
-import React, {Component} from 'react';
+import React, {useState, useEffect} from 'react';
```

Comment:
it's easier to look at this diff side-by-side -- there's a component to render this as an integrated example and i refactored it to a functional component because why not 🤷🏻‍♀️